### PR TITLE
refactor: made logger clearer

### PR DESCRIPTION
**Pull request by Ben-Cheng**
### Change Description:
Before:
image

After:
image

### CODE REVIEW

File: topo_processor/metadata/collection_factory.py [hunk 1]
```diff
@@ -13,16 +13,14 @@
...
 
-def create_item(collection: Collection, path: str) -> None:
-    start_time = time_in_ms()
+def create_items(collection: Collection, path: str) -> None:
     for f in os.listdir(path):
         file_path = os.path.join(path, f)
         item = Item(file_path, collection)
         repo.add_metadata(item)
         collection.items.append(item)
-    get_log().debug("CreateItem", path=path, duration=time_in_ms() - start_time)
```
Comment (blacha) [inline topo_processor/metadata/collection_factory.py:28]: the total time to add metadata is quite a useful log to have, why was this taken out?

File: topo_processor/metadata/item.py [hunk 2]
```diff
@@ -26,3 +27,4 @@ def __init__(self, path: str, collection: Collection):
             properties={},
             stac_extensions=[],
         )
+        get_log().info("Item initialised", path=self.path)
```
Comment (blacha) [inline topo_processor/metadata/item.py:30]: Once its been created its not actually initalised so this log is a little  missleading, I generally would avoid having logs in constructors too.

Comment (Ben-Cheng) [reply topo_processor/metadata/item.py:30]: why would you avoid having logs in constructors?

File: topo_processor/metadata/collection_factory.py (hunk 1 above)
Comment (Ben-Cheng) [reply topo_processor/metadata/collection_factory.py:28]: I have added it back. I think I removed it by mistake.

**Comment by Ben-Cheng** [comment]
Manually merged into the current feat/extract-info-from-tiff branch to avoid conflict

**Pull request closed without merging**